client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx

```diff
@@ -49,6 +51,32 @@ const BusinessPlanDetails = ( {
 		<div>
 			{ googleAppsWasPurchased && <GoogleAppsDetails isRequired /> }
 
+			{ isEnabled( 'jetpack/features-section' ) && (
+				<>
+					<PurchaseDetail
+						icon={ <img alt="" src={ jetpackBackupImage } /> }
+						title={ i18n.translate( 'Check your backups' ) }
+						description={ i18n.translate(
+							'Backup gives you granular control over your site, with the ability to restore it to any previous state, and export it at any time.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( 'Granular control over your site, with the ability to restore it to any previous state, and export it at any time.' )` *ES Score: 8*